tests/ui/perfherder/alerts-view/alerts_table_row_test.jsx

```diff
@@ -146,22 +156,11 @@ test("Button '...' displays all the tags for an alert item", async () => {
   expect(visibleTags).toHaveLength(testTags.length);
 });
 
-test("Alert item with no options displays 'No options'", async () => {
-  const { getByText } = alertTableRowTest({
-    alert: testAlert,
-    tags: false,
-    options: [''],
-  });
-
-  const message = await waitFor(() => getByText('No options'));
-  expect(message).toBeInTheDocument();
-});
-
 test('Alert item with 2 options displays 2 options', async () => {
   const testOptions = ['option1', 'option2'];
   const { getAllByTestId } = alertTableRowTest({
     alert: testAlert,
-    tags: false,
+    tags: [],
```

Comment:
I think that changing `tags: false` to `tags: []` should be done everywhere in `alerts_table_row_test.jsx`.

Reply:
I think the purpose of this is to not use the default tags that come from the json file.